tests/node/rpc/messages/UnsubscribeVehicleDataResponseTests.js

```diff
@@ -1,8 +1,14 @@
 const SDL = require('../../../config.js').node;
+
+// messages
 const UnsubscribeVehicleDataResponse = SDL.rpc.messages.UnsubscribeVehicleDataResponse;
+
+// enums
 const FunctionID = SDL.rpc.enums.FunctionID;
 const MessageType = SDL.rpc.enums.MessageType;
 const VehicleDataType = SDL.rpc.enums.VehicleDataType;
+
```

Comment:
**Optional:** On line 26 you could change `JSON_VEHICLEDATARESULT` to `JSON_HANDSOFFSTEERING`

Reply:
Based on previous experience, it is better to avoid changes not related to PR. Will add if PMs request this in the review.